spec/beaker-answers/pe_conf_spec.rb

```diff
@@ -21,6 +21,23 @@
       end
     end
 
+    context 'pe_postgres' do
```

Comment:
a) these specs don't pass

```

Failures:

  1) BeakerAnswers::PeConf 1.0 schema pe_postgres generates a hash of monolithic configuration data
     Failure/Error: if hosts[0].options[:pe_postgresql_options]

     NoMethodError:
       undefined method `options' for #<StringifyHash:0x007fc732b09040>
     Shared Example Group: "pe.conf configuration" called from ./spec/beaker-answers/pe_conf_spec.rb:89
     # ./lib/beaker-answers/pe_conf.rb:87:in `_generate_1_0_data'
     # ./lib/beaker-answers/pe_conf.rb:57:in `configuration_hash'
     # ./spec/beaker-answers/pe_conf_spec.rb:35:in `block (4 levels) in <top (required)>'

  2) BeakerAnswers::PeConf 2.0 schema pe_postgres generates a hash of monolithic configuration data
     Failure/Error: if hosts[0].options[:pe_postgresql_options]

     NoMethodError:
       undefined method `options' for #<StringifyHash:0x007fc731b81be0>
     Shared Example Group: "pe.conf configuration" called from ./spec/beaker-answers/pe_conf_spec.rb:131
     # ./lib/beaker-answers/pe_conf.rb:147:in `_generate_2_0_data'
     # ./lib/beaker-answers/pe_conf.rb:59:in `configuration_hash'
     # ./spec/beaker-answers/pe_conf_spec.rb:35:in `block (4 levels) in <top (required)>'

Finished in 0.92143 seconds (files took 0.24155 seconds to load)
224 examples, 2 failures

Failed examples:

rspec './spec/beaker-answers/pe_conf_spec.rb[1:1:2:1]' # BeakerAnswers::PeConf 1.0 schema pe_postgres generates a hash of monolithic configuration data
rspec './spec/beaker-answers/pe_conf_spec.rb[1:2:2:1]' # BeakerAnswers::PeConf 2.0 schema pe_postgres generates a hash of monolithic configuration data
```


b) You're not testing any of the conditional logic you added - please add tests for those

Reply:
doh.